### PR TITLE
Review docs

### PR DESCRIPTION
**Pull request by aadcg**
I've studied the library and made some minor corrections as I went through it.

### CODE REVIEW

File: .dir-locals.el [hunk 1]
```diff
@@ -0,0 +1,4 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+((nil . ((fill-column . 80))))
```
Comment (aartaka) [inline .dir-locals.el:4]: Actually, why are we setting it to 80? 100 is more customary in Lisp, due to long-ish names.

Anyway, that's a discussion for another time

File: README.org [hunk 2]
```diff
@@ -6,40 +6,38 @@ inspired by Emacs.
...
+  (e.g. "control-a control-b control-c").
 
-- A =keyspec= is a string representation of a =key=.  A =keyspec= is a
-  space-separated string representation of a sequence of =key=s.
-
-- A =keymap= contains a hash-table where the keys are a =keyspec= and the values
-  are either a keymap or anything else.  As a special case, a =nil= value
-  unbinds the key.
+- A =keymap= contains a hash-table that maps =keyspecs= to either keymaps,
+  symbols or nil (to unbind the =keyspecs=).
```
Comment (aartaka) [inline README.org:19]: Note that there can be other values that one can put there:
- `nyxt:command`s,
- `functions`s.
- And anything else, actually. `nkeymaps` is quite lenient about that.

So maybe we shouldn't mention it here and in the docstrings...

File: modifiers.lisp [hunk 3]
```diff
@@ -3,13 +3,13 @@
 
 (in-package :nkeymaps/modifier)
 
-(defparameter +control+ (nkeymaps:define-modifier :string "control" :shortcut "C")
+(defparameter +control+ (define-modifier :string "control" :shortcut "C")
   "The key modifier commonly known as 'control'.")
 (defparameter +meta+ (define-modifier :string "meta" :shortcut "M")
-  "The key modifier commonly known as 'meta' or 'alt'.")
+  "The key modifier commonly known as 'alt' or 'option'.")
```
Comment (aartaka) [inline modifiers.lisp:9]: I'd retain `meta` mention here, because that's still a vibrant echo of the times long gone 😛

**Review by aartaka — APPROVED** [review]

File: README.org [hunk 4]
```diff
@@ -6,40 +6,38 @@ inspired by Emacs.
...
-- A =key= structure has a hardware =code=, a symbolic =value=, an ordered set of
-  modifiers (the order is to avoid inequality on permutation) and possibly more
-  in the future.
+- A =key= structure that holds:
+  - a numeric =code= :: a hardware keycode;
+  - a string =value= :: a character (except for whitespaces);
+  - an ordered set of modifiers (to ensure equality on permutation).
 
-  Whitespaces are the only reserved characters.  Anything else can be stored in
-  the =key= value.
```
Comment (Ambrevar) [inline README.org:14]: The part about whitespace being reserved is important.

File: README.org [hunk 5]
```diff
@@ -6,40 +6,38 @@ inspired by Emacs.
 
 The main types are:
 
-- A =key= structure has a hardware =code=, a symbolic =value=, an ordered set of
-  modifiers (the order is to avoid inequality on permutation) and possibly more
-  in the future.
+- A =key= structure that holds:
+  - a numeric =code= :: a hardware keycode;
+  - a string =value= :: a character (except for whitespaces);
```
[the remark this answers is not in the document]
Comment (Ambrevar) [reply README.org:11]: OK!

**Comment by Ambrevar** [comment]
OK on my end.

**Pull request merged**